### PR TITLE
UI quiz screen

### PR DESCRIPTION
**Pull request by KuicsCristian**

### CODE REVIEW

File: Internship2022TeamSzabi/Internship2022TeamSzabi/Views/QuizViewController/QuizCustomCell/QuizQuestionTableViewCell.swift [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+//  Internship2022TeamSzabi
+//
+//  Created by Cristian Vasile Kui-cs on 08.08.2022.
+//
+
+import UIKit
+
+class QuizQuestionTableViewCell: UITableViewCell {
+    @IBOutlet private var buttonCollection: [UIButton]!
+    @IBOutlet weak var quizContainer: UIView!
```
Comment (iOSinternshipen) [inline Internship2022TeamSzabi/Internship2022TeamSzabi/Views/QuizViewController/QuizCustomCell/QuizQuestionTableViewCell.swift]: use private

**Review by iOSinternshipen — CHANGES_REQUESTED** [review]

**Review by iOSinternshipen — APPROVED** [review]

File: Internship2022TeamSzabi/Internship2022TeamSzabi/Views/QuizViewController/QuizViewController.swift [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+}
+
+extension QuizViewController: UITableViewDelegate, UITableViewDataSource {
+    func tableView(_ tableView: UITableView, numberOfRowsInSection section: Int) -> Int {
+        return 5
+    }
+    
+    func tableView(_ tableView: UITableView, cellForRowAt indexPath: IndexPath) -> UITableViewCell {
+        guard let cell = tableView.dequeueReusableCell(withIdentifier: QuizQuestionTableViewCell.identifier,
+                                                       for: indexPath) as? QuizQuestionTableViewCell else {
```
Comment (IuliaRAnisoi) [inline Internship2022TeamSzabi/Internship2022TeamSzabi/Views/QuizViewController/QuizViewController.swift:52]: should remove extra space before "for" provided that SwiftLint shows no error

File: Internship2022TeamSzabi/Internship2022TeamSzabi/Views/QuizViewController/QuizCustomCell/QuizQuestionTableViewCell.swift [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+class QuizQuestionTableViewCell: UITableViewCell {
+    @IBOutlet private var buttonCollection: [UIButton]!
+    @IBOutlet private weak var quizContainer: UIView!
+    
+    static let identifier = "QuizQuestionTableViewCell"
+    
+    static func nib() -> UINib {
+        return UINib(nibName: "QuizQuestionTableViewCell", bundle: nil)
+    }
+    override func awakeFromNib() {
```
Comment (IuliaRAnisoi) [inline Internship2022TeamSzabi/Internship2022TeamSzabi/Views/QuizViewController/QuizCustomCell/QuizQuestionTableViewCell.swift]: should add extra line of space here

File: Internship2022TeamSzabi/Internship2022TeamSzabi/Views/QuizViewController/QuizCustomCell/QuizQuestionTableViewCell.swift [hunk 4]
```diff
@@ -0,0 +1,35 @@
...
+        quizContainer.layer.cornerRadius = 20
+        quizContainer.layer.masksToBounds = false
+        buttonCollection.forEach {
+            $0.isSelected = false
+            $0.setImage(UIImage(named: "circle-unselected-icon"), for: UIControl.State.normal)
+            $0.setImage(UIImage(named: "circle-selected-icon"), for: UIControl.State.selected)
+        }
+    }
+    
+    @IBAction func buttonSelected(_ sender: UIButton) {
```
Comment (IuliaRAnisoi) [inline Internship2022TeamSzabi/Internship2022TeamSzabi/Views/QuizViewController/QuizCustomCell/QuizQuestionTableViewCell.swift]: this should be private

File: Internship2022TeamSzabi/Internship2022TeamSzabi/Views/QuizViewController/QuizViewController.swift [hunk 5]
```diff
@@ -0,0 +1,56 @@
...
+class QuizViewController: UIViewController {
+    @IBOutlet private weak var tableView: UITableView!
+    @IBOutlet private weak var finishButton: UIButton!
+    @IBOutlet private weak var fadeView: UIView!
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        
+        tableView.register(QuizQuestionTableViewCell.nib(),
+                           forCellReuseIdentifier: QuizQuestionTableViewCell.identifier)
```
Comment (IuliaRAnisoi) [inline Internship2022TeamSzabi/Internship2022TeamSzabi/Views/QuizViewController/QuizViewController.swift:19]: should remove extra space before argument name provided that SwiftLint shows no error

**Review by IuliaRAnisoi — COMMENTED** [review]
should delete the fixed height of the "Finish" button and instead select a "Large" size for it, from the Size Inspector (it's the first option)

**Review by IuliaRAnisoi — APPROVED** [review]

**Pull request merged**